src/actor.js

```diff
@@ -764,3 +764,40 @@ export const getApifyProxyUrl = (options = {}) => {
 
     return `http://${username}:${password}@${hostname}:${port}`;
 };
+
+/**
+ *
+ * Creates adhoc webhook for current actor run.
+ * Webkooks are supported only for runs on Apify cloud.
```

Comment:
`Webkooks`

Reply:
"Creates an ad-hoc webhook for the current actor run, which lets you receive a notification when the actor run finished or failed. For more information about Apify actor webhooks, please see the documentation.

Note that webhooks are only supported for actors running on the Apify platform. In local environment, the function will print a warning and have no effect."